### PR TITLE
Fix GitHub issue templates

### PR DESCRIPTION
**Pull request by drewnoakes**
#7476 added our a GitHub issue template for branch checklists. That PR used a newer method of specifying templates than the one we used previously.

Adding that template caused GitHub to show visitors the following when they try to create a new issue:

image

The indicated link would show our old template, but that is non-obvious.

This PR adds two new items to the list:

- Bug report
- Feature request

It also adds a link to the "Report a Problem" tool documentation.

## After

image

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/project-system/pull/7559)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Report an issue with the .NET Project System for Visual Studio
+title: 
+labels:
```
Comment (MiYanni) [inline .github/ISSUE_TEMPLATE/bug-report.md:5]: Please add `Bug` as an automatic label.

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+name: Feature request
+about: Request a new feature of .NET Project System for Visual Studio
+title: 
+labels:
```
Comment (MiYanni) [inline .github/ISSUE_TEMPLATE/feature-request.md:5]: Please add `Feature Request` as an automatic label.

**Review by MiYanni — APPROVED** [review]
Good idea. I think we'll end up using this for a bit until the [issue forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms) feature is out of beta.

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 1 above)
Comment (drewnoakes) [reply .github/ISSUE_TEMPLATE/bug-report.md:5]: I considered that but decided against it as I suspect we'll see a lot of stuff filed that isn't actually a bug. Let's do this in triage for things we actually believe are real bugs.

If that approach creates undue work I'm fine with adding the automatic label then.

File: .github/ISSUE_TEMPLATE/feature-request.md (hunk 2 above)
Comment (drewnoakes) [reply .github/ISSUE_TEMPLATE/feature-request.md:5]: Ditto.

**Pull request merged**